lib/select.c

```diff
@@ -209,7 +209,8 @@ int Curl_select(curl_socket_t maxfd,   /* highest socket number */
     descriptor set must contain at least one handle to a socket.
 
     It is unclear why WinSock doesn't just handle this for us instead of
-    calling this an error.
+    calling this an error. Luckily, with WinSock, we can _also_ ask how
+    many bits are set on an fd_set. So, let's just check it beforehand.
```

Comment:
This comment paragraph is unnecessary , it's clear what is being done by looking at the code.

Reply:
This was moved over from a previously existing comment, therefore I would rather like to keep it.